### ISSUE TITLE
关于fine-tuning步数的设置

### ISSUE THREAD

**Issue by YY-RR-ZZ** [huai-chang/DiffEIC#21]
作者您好，非常感谢您的开源工作，“we first train the proposed DiffEIC with λ = 1 for 300K iterations, and then adapt them using target λ
for another 200K iterations. We set the learning rate to 2×10−5 during the fine-tuning process.”在论文中是这么说的。我的问题是，比如我想训练lambda=4的模型，是不是先λ = 1跑300K次，然后λ = 4继续跑 200K次，这两阶段都是学习率1e-4，那fine-tuning process阶段学习率2×10−5得继续迭代多少次呢？

**Comment by huai-chang** [comment]
fine-tuning process 就是你说的将 $$\lambda$$ 设为4的阶段。训练分为两个阶段，第一个阶段使用 $$\lambda=1$$ 训练300k次，学习率为1e-4，第二阶段 $$\lambda$$ 根据需要从 {1,2,4,8,16} 中选取，训练200k次，学习率为2e-5。

**Comment by YY-RR-ZZ** [comment]
明白了，谢谢您
